src/CMakeLists.txt

```diff
@@ -73,7 +73,7 @@ lua_source(lua_sources ${LUAJIT_SOURCE_ROOT}/tools/utils/bufread.lua)
 lua_source(lua_sources ${LUAJIT_SOURCE_ROOT}/tools/utils/symtab.lua)
 
 add_custom_target(generate_lua_sources
-    WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/src/box
+    WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/src/box
```

Comment:
While we're here: why don't you use `${CMAKE_CURRENT_SOURCE_DIR}`/`${CMAKE_CURRENT_BINARY_DIR}` instead of `${PROJECT_SOURCE_DIR}/src`/`${PROJECT_BINARY_DIR}/src` within this CMakeLists.txt?

Reply:
`PROJECT_` one points to the top of `tarantool/`, while the `CURRENT_` one - to the `tarantool/src`, hence the 
```
    ${CMAKE_CURRENT_SOURCE_DIR}/third_party/crc32_impl.c
```
won't work. In case an `/src` subpath is used it might be ok, but we drift away from uniform use of variables.

Reply:
Yep, it doesn't work for `third_party` and I've proposed to use it only for the current directory instead of building this path by hands. Anyway, this is such a neglible comment, so feel free to ignore.